packages/flutter_tools/lib/src/package_map.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:package_config/packages_file.dart' show parse;
```

Comment:
perhaps prefix this? `parse` is pretty generic. `... as package_config show parse;`